deployments/helm/hephaestus/values.yaml

```diff
@@ -117,6 +117,10 @@ controller:
   # Additional volume mounts added to both the manager and vector containers
   extraVolumeMounts: []
 
+  ibGarbageCollector:
+    interval: "@daily"
+    maxIBRetention: 5
```

Comment:
Ditto, `maxIBRetention` isn't clear on its purpose - Forge had it as a `gc-max-keep` argument, maybe `keepMaximum: 5` or `retainMax: ...`?